paasta_tools/check_chronos_jobs.py

```diff
@@ -152,31 +144,65 @@ def message_for_status(status, service, instance, cluster):
         raise ValueError('unknown sensu status: %s' % status)
 
 
-def sensu_message_status_for_jobs(chronos_job_config, service, instance, cluster, job_state_pairs):
-    if len(job_state_pairs) > 1:
-        sensu_status = pysensu_yelp.Status.UNKNOWN
-        output = (
-            "Unknown: somehow there was more than one enabled job for %s%s%s.\n"
-            "Talk to the PaaSTA team as this indicates a bug." % (service, utils.SPACER, instance)
-        )
-    elif len(job_state_pairs) == 0:
+class TZ(tzinfo):
+
+    def utcoffset(self, dt):
+        return timedelta(minutes=0)
+
+    def dst(self, dt):
+        return timedelta(minutes=0)
+
+
+utc = TZ()
```

Comment:
Having this instance here outside of any method looks funny to me.

Reading the later comment... yuck :(

But I understand now. Time zones man :)